src/client/components/SearchMessagesWords/SearchMessages.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-shadow */
```

Comment:
Do not disable ES Lint warnings and fix each error individually.

Reply:
ok, i will fix it, thanks